test/route.test.js

```diff
@@ -859,7 +859,7 @@ test('HEAD route should handle properly each response type', t => {
   }, (error, res) => {
     t.error(error)
     t.equal(res.statusCode, 200)
-    t.equal(res.headers['content-type'], 'application/octet-stream')
+    t.same(res.headers['content-type'], null)
     t.equal(res.headers['content-length'], undefined)
```

Comment:
for consistency with how the assertion are made against content-length, let's use the same pattern for this header too. meaning, t.equal(..., undefined). here and everywhere else